tests/factories/reports/cht/delivery.js

```diff
@@ -0,0 +1,256 @@
+const Factory = require('rosie').Factory;
+const uuid = require('uuid');
+
+const defaultFields = {
```

Comment:
I think the fields need to be pulled into factories themselves. The fields are really what drives the data and from what I can tell Rosie will overwrite your nested objects if you try to do  `Factory.build('delivery', {'fields.pnc_danger_sign_check', 'blah'})`

Reply:
I actually think the better option is what is shown in the pregnancy factory. It would allow the user to define fields and a bunch of different preganancy factories

Reply:
If I understand you correctly, I agree....  so we'd have 1 "base" factory here, with lots of calls to `attr` to set up each little field, then a few others that extend the "base" one that serve as reuable convenience factories so people aren't having to copy/paste/change all over for the same cases.